### PR TITLE
Fixed commit message with chinese character (Close #286)

### LINKED ISSUE #286: Chinese comment not supported
- VSCode Version: 1.24.1 (1.24.1)
- OS Version: macOS 10.13.4
- Extension Version: 1.29.6
- System Language: English
- SVN Version: 1.9.7 (r1800392)

## Issue ##
can't commit chinese comment。
try add quotation marks when detected chinese?

## Steps to Reproduce ##

1. change some file
2. add comment and commit

## SVN Output ##
```
[comic]$ svn stat --xml --no-ignore --ignore-externals
[comic]$ svn cat src/html/search.html -r HEAD
[comic]$ svn update --ignore-externals
[comic]$ svn stat --xml --no-ignore --ignore-externals
[comic]$ svn info --xml
[comic]$ svn commit src/html/search.html -m 搜索历史长度截断
[comic]$ svn log -r BASE:HEAD -q --xml
svn: E000022: Commit failed (details follow):
svn: E000022: Error normalizing log message to internal format
svn: E000022: Can't convert string from native encoding to 'UTF-8':
svn: E000022: {U+641C}{U+7D22}{U+5386}{U+53F2}{U+957F}{U+5EA6}{U+622A}{U+65AD}
```

### PR DESCRIPTION
**Pull request by edgardmessias**
The problem is in decode characters in SVN, ignore defined environments.

To work, use a file with UTF-8 and pass the --encoding UTF-8 argument.

### CODE REVIEW

File: src/svnRepository.ts [hunk 1]
```diff
@@ -1,5 +1,7 @@
 import * as fs from "fs";
+import isUtf8 = require("is-utf8");
```
Comment (JohnstonCode) [inline src/svnRepository.ts]: This isn't used?

**Comment by edgardmessias** [comment]
@JohnstonCode, fixed

**Comment by edgardmessias** [comment]
I need fix the package-lock.json

**Comment by edgardmessias** [comment]
Done

**Comment by JohnstonCode** [comment]
Sorry to be a pain but can you prefix you commit message with ```fix: ``` < space after the `:`

This way CI will deploy it for me

**Comment by edgardmessias** [comment]
Done

**Pull request merged**